### DISCUSSION TITLE
Vue des WAN, MAN, LAN, et WLAN

### DISCUSSION THREAD

**Discussion by Guerisan** [dbarzin/mercator#270] [General]
Bonjour Didier,

C'est pour une petite précision concernant la vue dont font partie les WAN, MAN, LAN, et WLAN.
Ils font en principe  partie les infrastructures logiques (ils apparaissent sur cette vue), mais on doit les insérer depuis la catégorie des éléments physiques.

Je précise que je suis dans une quête pour modéliser l'intégralité des éléments dans l'ExplorerController, et c'est notamment pour être sûr de l'id de la vue à passer aux noeuds.

Peux-tu me confirmer qu'ils font bien partie des infrastructures logiques ? (auquel cas il faudra modifier le menu latéral)

Merci !

Bastien

**Comment by dbarzin** [comment]
Bonsoir Bastien,

C'est documenté comme cela dans le guide de l'ANSSI. Le VLAN aurait intérêt à être déplacé dans la vue de l'infrastructure logique car il y est clairement rattaché. Il n'a pas été attaché aux commutateurs physique comme le préconise l'ANSSI car souvent les VLANs sont déployés sur tous les commutateurs de l'infrastructure physique.

Quant aux WAN, MAN et LAN, comme ils ne sont liés qu'entre eux, cela doit encore être clarifié. Ma question est plutôt faut-il les conserver ?

Cordialement,
Didier

**Comment by Guerisan** [comment]
Je vois, question épineuse. Maintenant qu'ils sont là, l'utilisateur a toujours la liberté de s'en servir ou non.
